angular2/src/angular2TextMask.ts

```diff
@@ -28,8 +28,23 @@ export default class MaskedInputDirective implements ControlValueAccessor{
 
   formControl: FormControl = new FormControl()
 
-  constructor(inputElement: ElementRef) {
-    this.inputElement = inputElement.nativeElement
+  constructor(private element: ElementRef) {
+  }
+
+  ngAfterViewInit() {
+    if(this.element.nativeElement.tagName == 'INPUT'){
```

Comment:
Can you add a space after the `if` and use strict equality `===` for consistency?